engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-advanced-sync-options.html

```diff
@@ -0,0 +1,54 @@
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="standard" />
+      <span translate>
+        Optimized Sync
+      </span>
+    </label>
+    <div class="help-block">
+      <span translate>
+        Standard sync, optimized for speed by bypassing any unneeded steps.
```

Comment:
Should we make this the default, since it is what the user would have if they performed a sync without any advanced options?